library/x509.c

```diff
@@ -31,7 +31,7 @@
 
 /* Ensure gmtime_r is available even with -std=c99; must be included before
  * config.h, which pulls in glibc's features.h. Harmless on other platforms. */
-#define _XOPEN_SOURCE 500
+#define _XOPEN_SOURCE
```

Comment:
Apparently I was overzealous with request for this change. It turns out that `snprintf` requires one of the following:

    _BSD_SOURCE || _XOPEN_SOURCE >= 500 || _ISOC99_SOURCE || _POSIX_C_SOURCE >= 200112L

So unfortunately this change needs to be reverted.

Reply:
OK, I've rolled back that change.